apps/aecore/lib/aecore/peers/peer_connection.ex

```diff
@@ -593,7 +593,7 @@ defmodule Aecore.Peers.PeerConnection do
       port,
       share,
       genesis_hash,
-      :erlang.term_to_binary(difficulty),
+      :erlang.float_to_binary(difficulty),
```

Comment:
does epoch use `binary_to_float`?

Reply:
Yes, see [here](https://github.com/aeternity/epoch/blob/b724d07ebf5162586012ea54ed8e00eed9e74a88/apps/aecore/src/aec_peer_messages.erl#L174).

Reply:
thank you!